src/snmalloc/override/new.cc

```diff
@@ -71,3 +71,45 @@ void operator delete[](void* p, std::nothrow_t&)
 {
   ThreadAlloc::get().dealloc(p);
 }
+
+void* operator new(size_t size, std::align_val_t)
+{
+  return ThreadAlloc::get().alloc(size);
+}
+
+void* operator new[](size_t size, std::align_val_t)
+{
+  return ThreadAlloc::get().alloc(size);
+}
+
+void* operator new(size_t size, std::align_val_t, std::nothrow_t&)
+{
+  return ThreadAlloc::get().alloc(size);
+}
+
+void* operator new[](size_t size, std::align_val_t, std::nothrow_t&)
+{
+  return ThreadAlloc::get().alloc(size);
+}
+
+void operator delete(void* p, std::align_val_t)EXCEPTSPEC
```

Comment:
```suggestion
void operator delete(void* p, std::align_val_t) EXCEPTSPEC
```

I'm surprised `clang-format` was OK with that without the whitespace.

Reply:
... oh wow, it in fact requires that it not be there.  That... that sure seems like a bug, but I guess it's best to do what it wants.  Sorry!

Reply:
@nwf-msr sorry I am confused now.  I think @devnexen has now added the space, but are you saying the space shouldn't be there?

Reply:
by the way is clang-format9 still recommended ? Thankfully I still have it somewhere.

Reply:
It is, but I might move this and Verona forward to a newer version.  It is getting a bit long in the tooth now.